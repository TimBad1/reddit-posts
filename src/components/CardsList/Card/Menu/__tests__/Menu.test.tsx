// import React from 'react';
// import { shallow } from 'enzyme';
// import { Menu } from '../Menu';


// describe('Menu', () => {
//     test('should render', () => {
//         const wrapper = shallow(<Menu children={<div />} button={<button />}/>)
//         expect(wrapper).toBeDefined();
//         expect(wrapper.find('div.menu').isEmptyRender()).toBeFalsy();
//     })

//     test('should render (snapshot)', () => {
//         const wrapper = shallow(<Menu children={<div />} button={<button />}/>)

//         expect(wrapper).toMatchSnapshot()
//     })
// })

