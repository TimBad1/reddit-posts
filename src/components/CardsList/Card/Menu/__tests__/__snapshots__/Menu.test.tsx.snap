// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should render (snapshot) 1`] = `
<div
  className="menu"
>
  <button
    className="menuButton"
    onClick={[Function]}
  >
    <MenuIcon />
  </button>
</div>
`;
